index.js

```diff
@@ -90,8 +89,12 @@ function getCurrentGeneratorVersion() {
  * @exports rclnodejs
  */
 let rcl = {
-  _initialized: false,
-  _nodes: [],
+  
+  // flag identifies when module has been init'ed
+  initialized: false,
```

Comment:
I don't find this property has been used, I guess you mean |_initialized|?

Reply:
opps! my bad. Yes it should be _initialized. Fixed